### PR TITLE
cash_register.rb Adding cash_register file

### PR DESCRIPTION
**Pull request by danadaldos**
[#89114338]

### CODE REVIEW

File: objects/cash_register.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+
+# CashRegister class performs the functions of a cash register
+class CashRegister
+  def initialize(total = 0)
```
Comment (jasonnoble) [inline objects/cash_register.rb]: You should make this a float.  `total = 0.0`

File: objects/cash_register.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  def total
+    puts "$#{format('%.2f', @total)}"
+  end
+
+  def pay(tendered)
+    if tendered > @total
+      change = tendered - @total
+      puts "Your change is $#{format('%.2f', change)}"
+      @total -= tendered
```
Comment (jasonnoble) [inline objects/cash_register.rb]: If my total is $3.65, and I pay you $5.00, I should get $1.35 in change, and my new total should be 0.

File: objects/cash_register.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    elsif @total < 0
```
Comment (jasonnoble) [inline objects/cash_register.rb]: If you fix the @total above, this elsif can go away.

File: objects/cash_register.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      @total = 0
+
+    elsif @total > 0
```
Comment (jasonnoble) [inline objects/cash_register.rb]: If my total is $3.75 and I pay you $2.00, my new total should be $1.75, not $3.75.

File: objects/cash_register.rb (hunk 3 above)
Comment (danadaldos) [inline objects/cash_register.rb]: I tried to fix it, but I was not able to get the elsif to go away while also making rubocop happy.

**Pull request merged**